batch2/batch/batch.py

```diff
@@ -301,12 +330,26 @@ def getopt(obj, attr):
     job_id = record['job_id']
     id = (batch_id, job_id)
 
+    try:
+        body = await job_config(app, record)
+    except Exception as err:
+        status = {
+            'worker': None,
+            'batch_id': batch_id,
+            'job_id': job_id,
+            'user': record['user'],
+            'state': 'error',
+            'error': str(err),
```

Comment:
This should be traceback.format_exc(err), it has much more information, including backtrace and the exception chain.